src/core/screen-private.h

```diff
@@ -193,6 +194,8 @@ void          meta_screen_update_workspace_layout (MetaScreen             *scree
 void          meta_screen_update_workspace_names  (MetaScreen             *screen);
 void          meta_screen_queue_workarea_recalc   (MetaScreen             *screen);
 
+GdkMonitor* meta_screen_get_current_monitor (void);
```

Comment:
Why this is put here?  Can not it be put in somewhere in `src/ui`?

Reply:
What would be the difference? (honest question, I actually struggled trying to figure out where to put it)

Reply:
I simply think that it is best to limit code availability to place where it is needed. Why put in `screen.c` if it used only in ui? Otherwise there is no functional difference.